src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
The method `ColorSupplier.getRandomColor()` is being called statically. According to the checklist, we should avoid using static methods. It's better to create an instance of `ColorSupplier` and call the method on that instance.